src/ParlorDetail.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import App from './App.js';
+// import './css/ParlorDetail.css';
+import ParlorArea from './ParlorArea.js';
+import ParlorCard from './ParlorCard.js';
+
+class ParlorDetail extends Component {
+    constructor(props) {
+        super(props);
+    }
+    
+    render() {
+        return (
+        <div>
```

Comment:
We can probably restructure this with some more semantic HTML tags and restructuring